# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/BrowserChild.h
# Commit: 50aa16aa36fb
# Full Hash: 50aa16aa36fb3b808d4116377db79bdc6faaa6fe
# Author: Nika Layzell <nika@thelayzells.com>
# Date: 2020-04-08 03:36:50
# Regressor Bug: 1616353
# File Overlap Count: 1
# Description:
#   Bug 1616353 - Part 6.3: Replace many nsILoadContext mutators with assertions, r=smaug
#   
#   Many of these setters are redundant, and duplicate work done elsewhere. As it is
#   no longer correct to set nsILoadContext attributes at these times, they're being
#   replaced with assertions.
# ==============================================================================

diff -r c8c7d1ab9fb4 -r 50aa16aa36fb dom/ipc/BrowserChild.h
--- a/dom/ipc/BrowserChild.h	Tue Apr 07 21:39:14 2020 +0000
+++ b/dom/ipc/BrowserChild.h	Tue Apr 07 21:39:16 2020 +0000
@@ -749,9 +749,6 @@
 
   mozilla::ipc::IPCResult RecvAllowScriptsToClose();
 
-  mozilla::ipc::IPCResult RecvSetOriginAttributes(
-      const OriginAttributes& aOriginAttributes);
-
   mozilla::ipc::IPCResult RecvSetWidgetNativeData(
       const WindowsHandle& aWidgetNativeData);
 